idd/Energy+.idd.in

```diff
@@ -108483,27 +108483,116 @@ PythonPlugin:Instance,
        \note The class must inherit the EnergyPlusPlugin base class
 
 PythonPlugin:Variables,
-       \memo This object defines name identifiers for custom Python Plugin variable data that should be shared
-       \memo among all running Python Plugins.
-       \memo I will make this extensible later.
-       \unique-object
-       \min-fields 1
-   A1, \field Name
-       \required-field
-       \type alpha
-       \note This object name is purely for reporting and diagnostics
-   A2, \field Variable Name 1
-       \type alpha
-       \note This variable is used to identify and create a shared variable in EnergyPlus.
-       \note This variable can then be used by all running Python Plugin instances.
-   A3, \field Variable Name 2
-       \type alpha
-   A4, \field Variable Name 3
-       \type alpha
-   A5, \field Variable Name 4
-       \type alpha
-   A6; \field Variable Name 5
-       \type alpha
+        \memo This object defines name identifiers for custom Python Plugin variable data that should be shared
+        \memo among all running Python Plugins.
+        \unique-object
+        \min-fields 2
+        \extensible:1
+   A1,  \field Name
+        \required-field
+        \type alpha
+        \note This object name is purely for reporting and diagnostics
+   A2,  \field Variable Name 1
+        \type alpha
+        \begin-extensible
+        \note This variable is used to identify and create a shared variable in EnergyPlus.
+        \note This variable can then be used by all running Python Plugin instances.
+   A3,  \field Variable Name 2
+        \type alpha
+   A4,  \field Variable Name 3
```

Comment:
Since this is now extensible, I think we can actually stop at just 1...or is preferred to go ahead and flesh out a decent amount?  IDF editor reasons?  @mjwitte @JasonGlazer?

Reply:
I'd probably put in at least ten copies of the extensible field just for IDF Editor.

Reply:
OK, that has been updated to leave 10 copies.

Reply:
Thanks @mitchute this is a great fix.